hazelcast/src/main/java/com/hazelcast/core/ClientListener.java

```diff
@@ -19,22 +19,22 @@
 import java.util.EventListener;
 
 /**
- * The ClientListener provides the ability to listen to clients connecting and disconnecting from the cluster.
+ * The ClientListener provides the ability to listen to clients connecting and disconnecting from the member.
  *
  * @see Client
  * @see ClientService#addClientListener(ClientListener)
  */
 public interface ClientListener extends EventListener {
 
     /**
-     * Invoked when a new client is connected.
+     * Invoked when a client is connected.
```

Comment:
Will it be one if client uses 3 connections to the member? Do we need this level of documentation?